### ISSUE TITLE
loc("Power_3939"): error: SCALARS are not supported

### ISSUE THREAD

**Issue by wocsor** [luxonis/depthai-ml-training#45]
Hi, I'm trying to compile an IR model which was converted from ONNX using openvino 2022.1 and I'm getting a compilation error. The device is running LuxonisOS 1.5. The model is FP16 and I have not run it through the POT. Model binary is below.

model onnx: https://github.com/commaai/openpilot/raw/1e49c54ffb274a7987626ebfaa8eb3e75ac6fe7c/selfdrive/modeld/models/supercombo.onnx

```
root@keembay:~# /opt/openvino/tools/compile_tool/compile_tool -d VPUX.3400 -m supercombo.xml -ip FP16 -ov_api_1_0
OpenVINO Runtime version ......... 2022.1.0
Build ........... 2022.1.0-7080-6582ec65d78-releases/2022/1
Network inputs:
    big_input_imgs : FP16 / NCHW
    desire : FP16 / CHW
    features_buffer : FP16 / CHW
    input_imgs : FP16 / NCHW
    traffic_convention : FP16 / NC
Network outputs:
    outputs : FP32 / NC

Callback signal handler installed

Opening XLink Device File
loc("Power_3939"): error: SCALARS are not supported
Compilation failed
```

**Comment by tersekmatija** [comment]
Hey @wocsor, seems that the model is stored with GitLFS. Do you mind sharing a downloadable file directly please?

**Comment by wocsor** [comment]
Had to upload it to Google Drive - here's a link. I also went ahead and included the xml and bin files from openvino 2022.1 mo.py 
https://drive.google.com/drive/folders/1ZS6Bh1K8h81b1bSciqal70SlR9k36SBN?usp=sharing

**Comment by tersekmatija** [comment]
Hey, thanks! I've checked the model. It contains ReduceL2 which is not supported at the moment, which causes the model to fail. Since the output of the branch that includes ReduceL2 is then fused with other inputs I think it might be hard to prune it before this operation. We are working on providing support for more operations, but no ETA at the moment - we are hoping to provide it in the following few weeks.

**Comment by wocsor** [comment]
Is there any update? I saw someone else mentioned this issue on Discord as well.

**Comment by tersekmatija** [comment]
Hey, unfortunately not. One possible workaround for it would be to split the model into two parts - before and after the unsupported operation. In the middle then you could use the script node and perform the L2 manually, then feed this as an input to the second part.

**Comment by tersekmatija** [comment]
Hey @wocsor , 
Happy to share that it's now possible to export the supercombo model (files you've shared) with the latest [dev-blobconverter](https://dev-blobconverter.luxonis.com/).

**Comment by wocsor** [comment]
yep, just tested and it is outputting a blob file! now just need to test it on the hardware. is the conversion process updated? if so, are there instructions available for building the blobs locally?

**Comment by tersekmatija** [comment]
@wocsor 
I'd recommend using this for the time being to have the access to the latest compiler. In the future we might add a new version of compile tool to the latest LuxonisOS or expose it in some other way - either by releasing the dev-blobconverter or providing a containerized environment with the latest compiler. Will keep you updated soon.

**Comment by tersekmatija** [comment]
@wocsor 
You should be able to use [this branch of blobconverter](https://github.com/luxonis/blobconverter/tree/rvc3). It should match the latest [dev-blobconverter.luxonis.com](https://dev-blobconverter.luxonis.com/). Please refer to the `Prepare` section first, then you can use docker compose to build and deploy it locally. Note that the image can get quite big due to older RVC2 OpenVINO versions.
